### PR TITLE
Task 108196: cypress tags for filtering tests

### PR DESCRIPTION
**Pull request by Lisa-V-Sau**
Cypress grep:
- Dependency **cypress 10.0^** 
- Grep Module import to /plugins/index.js & /support/e2e.js
- Tags in use: @dev,@ stage, @integration
- For more recipes: https://github.com/cypress-io/cypress-grep

Cypress upgrade:
- Cypress package version: **10.9.0**
- Folder where tests live is renamed: **from 'integration', to 'e2e'**
- Test files renamed: **from '.spec.js', to '.cy.js'**
- Support files renamed: from **'/support/index.js', to 'support/e2e.js'**
- Cypress config files changed: **from 'cypress.json', to 'cypress.config.ts'**
- Latest changes: https://docs.cypress.io/guides/references/changelog

Cypress 10.9.0 known issues:
- Unable to run all tests at once via Cypress GUI: https://github.com/cypress-io/cypress/discussions/21628
- Error Log each time tests run in terminal: https://github.com/cypress-io/cypress/issues/22273

README updates:
- grep run command
- grep recipes

Files removed:
- Screen size import which was initially a spike is now outdated and is removed

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90893_SmokeTestSetUpExport.js [hunk 1]
```diff
@@ -1,38 +0,0 @@
-/// <reference types ='Cypress'/>
-// Guide: The exported smoke test will run within the screeSizeImport.js File.
-// Step1: To use this effectively include your test commands into this file without the 'Before' & 'After' Hooks.
-// Step2: Ensure to include notes for each test so that changes can be updated easily as needed for each smoke run. Example is provided below.
-
-export const smokeTest = () => {
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90893_SmokeTestSetUpExport.js:6]: Why are these tests being removed?

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90892_ScreenSizeTestImport.js [hunk 2]
```diff
@@ -1,13 +0,0 @@
-/// <reference types ='Cypress'/>
```
[the remark this answers is not in the document]
Comment (Lisa-V-Sau) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90892_ScreenSizeTestImport.js:1]: This was a spike for applying screen size's and applying them to a subset of tests which are outdated and no longer in need.

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90893_SmokeTestSetUpExport.js (hunk 1 above)
Comment (Lisa-V-Sau) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/90888_Automation_AdHoc_Smoke_Screen-Size/90893_SmokeTestSetUpExport.js:6]: This was a spike for applying screen size's and applying them to a subset of tests which are outdated and no longer in need.

File: README.md [hunk 3]
```diff
@@ -112,3 +118,18 @@ before(function () {
 ```
 
 Further details about Cypress can be found here: https://docs.cypress.io/api/table-of-contents
+
+To run tests with multiple tags:
+
+```
+i.e., greTags=@dev+@stage
```
[the remark this answers is not in the document]
Comment (Lisa-V-Sau) [reply README.md:125]: Made a change for this.

**Review by jordan-beard — APPROVED** [review]

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**